iothub/device/src/InternalClient.cs

```diff
@@ -861,19 +826,17 @@ public async Task SetMethodDefaultHandlerAsync(MethodCallback methodHandler, obj
         public async Task SetMethodDefaultHandlerAsync(MethodCallback methodHandler, object userContext, CancellationToken cancellationToken)
         {
             if (Logging.IsEnabled)
-            {
                 Logging.Enter(this, methodHandler, userContext, nameof(SetMethodDefaultHandlerAsync));
-            }
 
             cancellationToken.ThrowIfCancellationRequested();
-            await _methodsDictionarySemaphore.WaitAsync(cancellationToken).ConfigureAwait(false);
+            await _methodsSemaphore.WaitAsync(cancellationToken).ConfigureAwait(false);
 
             try
             {
                 if (methodHandler != null)
                 {
                     // codes_SRS_DEVICECLIENT_10_005: [ It shall EnableMethodsAsync when called for the first time. ]
```

Comment:
the comment still has the old name (same for other lines). Not sure what these codes_*  lines are, but should they be removed?

Reply:
I'll remove these specific ones. Yeah, I think we should probably remove (or at least refactor many of these to normal code comments. I'm going to pass on doing that at this time, however.